src/App.tsx

```diff
@@ -1,7 +1,42 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  wasPresed: boolean;
```

Comment:
Why do you need this?
If you have a null in `keyPres` you know that no one pressed a button, no?
```suggestion
```